sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -402,6 +402,9 @@ static struct snd_soc_dai_link dailink[] = {
 };
 
 /* SoC card */
+#if !IS_ENABLED(CONFIG_SND_SOC_INTEL_USER_FRIENDLY_LONG_NAMES)
+static char sdw_card_long_name[40]; /* sof-sdw-rt711-mono/stereo-rt1308-rt715 */
```

Comment:
I think it's pretty safe to assume, that "stereo" is longer than "mono" so you could do
```
/* can be "sof-sdw-rt711-stereo-rt1308-rt715" or "sof-sdw-rt711-mono-rt1308-rt715" */
static char sdw_card_long_name[] = "sof-sdw-rt711-stereo-rt1308-rt715";
```
to avoid a fixed size array and making sure it's future safe.

Reply:
@lyakh Done, thanks.